### PR TITLE
add functionality to delete event from production

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

- functionality to delete event from production

screenshot:
Schermafbeelding 2020-08-14 om 16 16 02

---
Ticket: https://jira.uitdatabank.be/browse/III-3304

### CODE REVIEW

File: pages/manage/productions/index.vue [hunk 1]
```diff
@@ -66,6 +80,8 @@
 
         isLoadingEvents: true,
         events: [],
+        toBeDeletedEventId: '',
+        toBeDeletedEventName: '',
```
Comment (geoffreydhuyvetters) [inline pages/manage/productions/index.vue:84]: can't we avoid this by passing the id to the modal as discussed? 

delete -> open modal with event information and production passed 
confirm -> trigger action back in index with the eventId passed down

Comment (simon-debruijn) [reply pages/manage/productions/index.vue:84]: yes, I could only pass the id down to the modal and then search for the eventById in the modal itself to retrieve the name. but toBeDeletedEventId still would have to be a data prop in index no?

Is this what you mean, or is it something different?

Comment (geoffreydhuyvetters) [reply pages/manage/productions/index.vue:84]: No, I don't think it needs to be, we can communicate through events from the modal. But do correct me if I'm oversimplifying it?

File: pages/manage/productions/index.vue [hunk 2]
```diff
@@ -26,9 +29,18 @@
             :has-adding-error="hasAddingEventToProductionError"
             @addEventToProduction="handleAddEventToProduction"
             @inputEventId="handleInputEventId"
+            @clickDeleteEvent="handleClickDeleteEvent"
           />
+          <delete-modal
+            :production-name="selectedProductionName"
+            :event-name="toBeDeletedEventName"
+            @confirmDelete="handleConfirmDeleteEvent"
```
Comment (geoffreydhuyvetters) [inline pages/manage/productions/index.vue:37]: I think @confirm is ok here since we're in the delete modal

**Pull request merged**